docs/getting_started.md

```diff
@@ -60,6 +60,8 @@ The variable group should contain the following required variables:
 | WORKSPACE_NAME           | mlops-AML-WS             |
 | WORKSPACE_SVC_CONNECTION | aml-workspace-connection |
 | ACI_DEPLOYMENT_NAME      | diabetes-aci             |
+| DATASET_NAME      | diabetes_training             |
```

Comment:
this should be optional, please remove.

Reply:
Why optional? One way or another, they'll need to create a dataset, won't they?

Reply:
See update